app/controllers/pruebas_controller.rb

```diff
@@ -0,0 +1,75 @@
+class PruebasController < ApplicationController
+  before_action :set_prueba, only: [:show, :edit, :update, :destroy]
```

Comment:
You can use ‘except:’ instead
Also this can be rewrites using ‘%w’
I recommend you to implement Rubocop as a ruby lint. Please table a look at: 
https://github.com/rubocop-hq/rubocop